.yamato/ruamel/jobs/abv/abv_all_project_ci_nightly.py

```diff
@@ -14,9 +14,14 @@ def __init__(self, editor, projects, test_platforms, nightly_config, target_bran
     def get_job_definition(self, editor, projects, test_platforms, extra_dependencies, target_branch): 
 
         # define dependencies
-        dependencies = [{
+        dependencies = [
+            {
                 'path': f'{abv_filepath()}#{abv_job_id_all_project_ci(editor["track"])}',
-                'rerun': editor["rerun_strategy"]}]
+                'rerun': editor["rerun_strategy"]},
+            {
+                'path': f'{templates_filepath()}#{template_job_id_test_all(editor["track"])}',
+                'rerun': editor["rerun_strategy"]
```

Comment:
i suspect the changes to the yml and py files are the result of a bad merge - please resolve this before landing to master

Reply:
Thanks @sophiaaar that was a bad merge indeed.
I fixed the merge in revision cd8d544
Those .yml and .pm [file changes](https://github.com/Unity-Technologies/Graphics/pull/1616/files) do not appear anymore.

Reply:
thank you! approved for this